distributed/comm/__init__.py

```diff
@@ -0,0 +1,12 @@
+from __future__ import print_function, division, absolute_import
+
+from .core import connect, listen, CommClosedError
+# Register transports
+from . import tcp
+try:
+    import zmq as _zmq
+except ImportError:
+    pass
+else:
+    # XXX this registers the ZMQ event loop, event if ZMQ is unused...
+    from . import zmq
```

Comment:
So, now the question is how we test it... Do we use an environment variable, some kind of configuration option? I'm not sure we can safely change IOLoops during the test suite...